apps/datahub/src/app/dataset/dataset-information/dataset-information.component.html

```diff
@@ -18,9 +18,12 @@
       <span class="font-semibold" translate="">mel.dataset.categories</span>
       <div class="flex gap-1 gap-x-2 flex-row flex-wrap">
         @for(topic of displayCategories; track $index) {
-        <span class="mel-badge-primary truncate">
+        <button
+          class="mel-badge-primary truncate"
+          (click)="onInfoKeywordClick(topic)"
```

Comment:
I wouldn't use the terms `keyword` nor `topic` here to prevent confusion with those, but rather `categoryKeyword` or `category`.